salt/metalk8s/map.jinja

```diff
@@ -2,13 +2,30 @@
 {% import_json 'metalk8s/versions.json' as versions with context %}
 
 {# First merge the basic defaults with versions information #}
-{% set version_defaults = {
-  'kubernetes': versions.kubernetes,
-  'repo': {
-    'packages': versions.packages,
-    'images': versions.images,
+{% set version_defaults = salt['grains.filter_by']({
+  'CentOS': {
+    'kubernetes': versions.kubernetes,
+    'repo': {
+      'packages': versions.packages.centos,
+      'images': versions.images,
+    },
+  },
+  'Ubuntu': {
+    'kubernetes': versions.kubernetes,
+    'repo': {
+      'packages': versions.packages.ubuntu,
+      'images': versions.images,
+    },
   },
-} %}
+}, grain='os', merge=defaults.get('version_defaults')) %}
```

Comment:
I suggest you use `os_family` instead of `os` as we expect that most of the code will work for RedHat and CentOS

Reply:
PS: If you give no `grain` arg it's `os_family` by default https://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.grains.html#salt.modules.grains.filter_by